engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/puppet-modules/content-view-puppet-modules.controller.js

```diff
@@ -26,7 +26,10 @@ angular.module('Bastion.content-views').controller('ContentViewPuppetModulesCont
         $scope.versionText = function (module) {
             var version;
             if (module['computed_version']) {
-                version = translate("Latest (Currently %s)").replace('%s', module['computed_version']);
+                if (module['uuid'])
+                    version = translate("Currently %s").replace('%s', module['computed_version']);
+                else
+                    version = translate("Latest (Currently %s)").replace('%s', module['computed_version']);
```

Comment:
I haven't tested this out yet, but normally in javascript it can be dangerous to use if statements without curly braces ({}).  I would suggest using those:
```
if (module['uuid']) {

} else {

}
```